Jenkinsfile-pr

```diff
@@ -37,6 +37,7 @@ pipeline {
         TEST_CLUSTER_ADMIN = "admin"
         OPERATOR_IMAGE_PULL_POLICY = "IfNotPresent"
         COMPONENTS_IMAGE_PULL_POLICY = "IfNotPresent"
+        JAVA_VERSION = "11"
```

Comment:
shouldn't we keep this on version `1.8` until the migration to java11 ?

Reply:
Is this question or suggestion? :)

Reply:
Kinda both :)